### DISCUSSION TITLE
Occasional messages failing to send, but OK on repeat. Ideas?

### DISCUSSION THREAD

**Discussion by colinb1950** [PHPMailer/PHPMailer#2848] [Q&A]
I have written a web application to manage my club membership and have added a function to send our renewal reminder emails. With less than 50 members I have a simple loop to send out mails one by one rather than as a bulk send. The message is boilerplate text with 2 data fields embedded in it for each menu.

When I run it it goes fine for maybe 10 or so (number varies), and then I get a message 'Language string failed to load: data_not_accepted'. But when I run the program again, that message gets sent without any issues. 

Any ideas why a simple text message would generate an error, but run OK the next time?

Heres my code
```php
                    $mail = new PHPMailer();
                    $mail->SetLanguage("en");
                    $headers='';
                    $mail->IsSMTP();                                                      
                    $mail->Host = "xxxxxxx";    
                    $mail->SMTPAuth = true;                                        
                    $mail->Username = "xxxxxxx@xxxxxxx";    
                    $mail->Altname = "xxxxxxx@xxxxxxx";       
                    $mail->Password = "xxxxxxx";                                         
                    $mail->From = $mail->Altname;	                                       
                    $mail->FromName = "Membership";

                    $mail->AddAddress($memberemail,$memberfullname);
                    $mail->AddCC($secretaryemail,$secretaryname);        

                    $mail->WordWrap = 50;                                                  
                    $mail->IsHTML(true);                                                   
                    $headers .= "MIME-Version: 1.0\r\n";
                    $headers .= "Content-Type: text/html; charset=utf-8\r\n";
                    $mail->CharSet = 'utf-8';

                    $mail->Subject = "Membership Renewals";
 
                    $mail->Body    = $message;

                    if(!$mail->Send()) {
                        echo "Message could not be sent. <p>";
                        echo "Mailer Error: " . $mail->ErrorInfo;
                        exit;
                    } else {
                        echo "Messages sent OK<br>";
                    }
```

**Comment by Synchro** [comment]
There are several things wrong here that won't help. You're using a _very_ old version of PHPMailer that's no longer supported, and you've based your code on a similarly old example. The "language string failed to load" issue was fixed many years ago.

It's likely that you're running into a send limit on your mail server; that would cause a "data not accepted" error.

You're setting some standard headers in `$headers`, but you don't need to do that – PHPMailer takes care of all that for you.

I suggest you update to the latest version, and start again basing your code on [the mailing list example provided](https://github.com/PHPMailer/PHPMailer/blob/master/examples/mailing_list.phps).

**Reply by colinb1950** [reply]
Many thanks
